src/pages/workspace/WorkspaceInvitePage.tsx

```diff
@@ -120,12 +121,16 @@ function WorkspaceInvitePage({
         });
 
         const newSelectedOptions: MemberForList[] = [];
-        Object.keys(invitedEmailsToAccountIDsDraft ?? {}).forEach((login) => {
-            if (!(login in detailsMap)) {
-                return;
-            }
-            newSelectedOptions.push({...detailsMap[login], isSelected: true});
-        });
+        if (firstRenderRef.current) {
+            // We only want to add the saved selected user on first render
+            firstRenderRef.current = false;
+            Object.keys(invitedEmailsToAccountIDsDraft ?? {}).forEach((login) => {
+                if (!(login in detailsMap)) {
+                    return;
+                }
+                newSelectedOptions.push({...detailsMap[login], isSelected: true});
+            });
+        }
```

Comment:
I decided to use `firstRenderRef` here instead of this init in `useState`.

```
const [selectedOptions, setSelectedOptions] = useState<MemberForList[]>(() => Object.values(invitedEmailsToAccountIDsDraft ?? {})
    .filter((accountID) => personalDetailsProp?.[accountID])
    .map((accountID) => ({...OptionsListUtils.formatMemberForList(personalDetailsProp[accountID]), isSelected: true}))
);
```

The reason is because the personal detail data of `personalDetailsProp` is not the same as  `inviteOptions.personalDetails`.

I first think of copying this logic:
```
const inviteOptions = OptionsListUtils.getMemberInviteOptions(personalDetailsProp, betas ?? [], searchTerm, excludedUsers, true);
// Update selectedOptions with the latest personalDetails and policyMembers information
const detailsMap: Record<string, MemberForList> = {};
inviteOptions.personalDetails.forEach((detail) => {
    if (!detail.login) {
        return;
    }
    detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail);
});
```
but I think it would be better to just use the ref.